test/shared/box.lua

```diff
@@ -22,7 +22,7 @@ local _TARANTOOL_MINOR = parse_tarantool_version(2)
 local _TARANTOOL_PATCH = parse_tarantool_version(3)
 local _TARANTOOL_REV = parse_tarantool_version(4)
 
-local function tarantool_version_at_least(major, minor, patch, rev)
+function tarantool_version_at_least(major, minor, patch, rev)
```

Comment:
I would maybe expose it where other functions (`test_1`. `test_2`, ...) are exposed using `_G.tarantool_version_at_least = tarantool_version_at_least`. It would be unexpected from the block of utility functions to expose a global function. Not sure. Up to you.

Reply:
fixed